lib/purchases_flutter.dart

```diff
@@ -946,8 +961,20 @@ class Purchases {
   }
 }
 
+class ProductChangeInfo {
+  /// The oldProductIdentifer to change from.
+  String oldProductIdentifer;
+
+  /// The [ProrationMode] to use when changing from the given oldProductIdentifer.
+  ProrationMode? prorationMode;
+
+  /// Constructs an ProductChangeInfo
+  ProductChangeInfo(this.oldProductIdentifer, {this.prorationMode});
+}
```

Comment:
I know UpgradeInfo is in this file, but now that we are making this from scratch I think it's a good opportunity to create a `product_change_info.dart` to hold this

Reply:
Aye aye, can do that 💪